docs/user-guide/org-settings.md

```diff
@@ -31,6 +31,10 @@ Sent invites can be invalidated by pressing the trash button in the relevant _Pe
 
 This tab displays an org's current payment plan and associated quotas along with usage history statistics for previous months.
 
+## Crawl Workflows
+
+This tab lets admins change the default suggested values for new crawl workflows. These defaults are not enforced and users can set their own values, or revert them back to the server-set defaults and limits.
```

Comment:
Similar feedback here, for more user-centered docs in active voice, with less punitive language (e.g. "not enforced"):

`Set and update default suggested settings for new crawl workflows. When creating a new workflow, org members will see that the form is pre-filled with default values. Org members can still change or remove these settings when configuring the workflow. Removing a default setting will revert the setting back to Browsertrix defaults.`

Also, I don't think we need to mention that they're not enforced since we say upfront that they're suggested.